playbooks/configure/hosts

```diff
@@ -0,0 +1 @@
+localhost ansible_connection=local
```

Comment:
All the tasks are run on `localhost` and there is only one play in the playbook, so you could move the `ansible_connection` option as an option of the play. Then, you could remove the `hosts` file and run the playbook with `ansible-playbook -i localhost, main.yml`.

Reply:
could do, but defining in a hosts file makes it applicable to all plays which I thought might be more versatile?

Reply:
Then the `hosts` attribute of the plays could be `all` rather than `localhost`. That would allow you to run the playbook from another machine than CloudForms appliance.